Lib/subprocess.py

```diff
@@ -733,6 +737,14 @@ def __init__(self, args, bufsize=-1, executable=None,
                 raise ValueError("preexec_fn is not supported on Windows "
                                  "platforms")
         else:
+            if _vxworks:
+                if shell:
+                    raise ValueError("shell is not supported on VxWorks")
+                if preexec_fn is not None:
+                    raise ValueError("Preexecution function is not supported"
```

Comment:
just name the argument in the ValueError message.  preexec_fn here, start_new_session below, etc.